deploy/cloudformation/parity_test.js

```diff
@@ -0,0 +1,7 @@
+const Web3 = require('web3');
+const web3 = new Web3(new Web3.providers.HttpProvider("http://ethereum.services.orbs-test.com:8545"));
```

Comment:
Every federation member will have its own URL, so this should be parametrized as well.

Reply:
We decided it's going to be a part of different effort when we are preparing for alpha release.